inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/config/InstrumentationConfigurationResolver.java

```diff
@@ -120,6 +121,10 @@ private boolean isIgnoredClass(Class<?> clazz, InstrumentationConfiguration conf
             return true;
         }
 
+        if (DoNotInstrumentMarker.class.isAssignableFrom(clazz)) {
```

Comment:
I don't see a test update for this? Or we don't even have the test for the resolver?